src/content/blog/mux-official-video-partner.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Mux: Our Official Video Partner"
+authors:
+  - fred
+description: |
+  We are happy to announce Mux as our official video partner, donating $5,000 each month toward the ongoing open-source maintenance and development of Astro.
+homepageLink:
+  title: Astro ❤️ Mux
+  subtitle: Our New Official Video Partner
+publishDate: "September 9, 2025"
+socialImage: "/src/content/blog/_images/mux-official-video-partner/astro-mux-social.webp"
+coverImage: "/src/content/blog/_images/mux-official-video-partner/astro-mux-header.webp"
+lang: "en"
```

Comment:
```suggestion
lang: "en"
related:
  - codetv
```
This might break the build without the other post being published yet, (and this is a case study slug, so hopefully that works here) so do not commit this yet!

Reply:
Shoudl work now